variables.tf

```diff
@@ -9,73 +10,19 @@ variable "common_tags" {
   default     = {}
 }
 
-variable "fifo_queue" {
-  description = "Is this queue fifo?"
-  type        = bool
-  default     = true
-}
-
-variable "api-token" {
+variable "api_token" {
   type        = string
   description = "Bridgecrew Platform API key"
-  sensitive   = true
```

Comment:
@mikeurbanski1 any reason not to mark it as sensitive?

Reply:
The attribute is not supported before TF 14.